opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/exponential/aggregation.py

```diff
@@ -0,0 +1,555 @@
+                self._positive.offset, self._positive._counts
+            ),
+            negative=BucketsPoint(
+                self._negative.offset, self._negative._counts
+            ),
+            # FIXME: Find the right value for flags
+            flags=0,
+            min=min_,
+            max=max_,
+        )
+
+        if self._previous_point is None or (
+            self._instrument_temporality is aggregation_temporality
+        ):
+            self._previous_point = current_point
+            return current_point
+
+        max_ = current_point.max
+        min_ = current_point.min
+
+        if aggregation_temporality is AggregationTemporality.CUMULATIVE:
+            start_time_unix_nano = self._previous_point.start_time_unix_nano
+            sum_ = current_point.sum + self._previous_point.sum
+            # Only update min/max on delta -> cumulative
+            max_ = max(current_point.max, self._previous_point.max)
+            min_ = min(current_point.min, self._previous_point.min)
+
+            negative_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.negative.bucket_counts,
+                    self._previous_point.negative.bucket_counts,
+                )
+            ]
+            positive_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.positive.bucket_counts,
+                    self._previous_point.positive.bucket_counts,
+                )
+            ]
+        else:
+            start_time_unix_nano = self._previous_point.time_unix_nano
+            sum_ = current_point.sum - self._previous_point.sum
+
+            negative_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.negative.bucket_counts,
+                    self._previous_point.negative.bucket_counts,
+                )
+            ]
+            positive_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.positive.bucket_counts,
+                    self._previous_point.positive.bucket_counts,
+                )
+            ]
+
+        current_point = ExponentialHistogramDataPoint(
+            attributes=self._attributes,
+            start_time_unix_nano=start_time_unix_nano,
+            time_unix_nano=current_point.time_unix_nano,
+            count=(
+                sum(negative_counts) + sum(positive_counts) + self._zero_count
+            ),
+            sum=sum_,
+            scale=self._scale,
+            zero_count=self._zero_count,
+            positive=BucketsPoint(self._positive.offset, positive_counts),
+            negative=BucketsPoint(self._negative.offset, negative_counts),
+            # FIXME: Find the right value for flags
+            flags=0,
```

Comment:
I don't think you need to worry about flags here, there are no specified uses for SDKs at this time.